### PR TITLE
Add RHEL9 support

### PR DESCRIPTION
**Pull request by jsafrane**
Add support for RHEL 9.

RHEL 9 does not ship python2 and libcgroup. I am able to create a RHEL9 cluster with this PR, I did not check any older distro.

### CODE REVIEW

File: nodeup/pkg/model/packages.go [hunk 1]
```diff
@@ -61,7 +61,6 @@ func (b *PackagesBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 		c.AddTask(&nodetasks.Package{Name: "ebtables"})
 		c.AddTask(&nodetasks.Package{Name: "ethtool"})
 		c.AddTask(&nodetasks.Package{Name: "iptables"})
-		c.AddTask(&nodetasks.Package{Name: "libcgroup"})
```
Comment (hakman) [inline nodeup/pkg/model/packages.go:64]: Let's add a case for RHEL 9 and move the `libcgroup` to `default` case to keep behaviour for existing distros.

Comment (jsafrane) [reply nodeup/pkg/model/packages.go:64]: I removed `switch`, there would be too much duplication (i.e. install libcgroup in `DistributionAmazonLinux2023:` in in `default:`)

File: nodeup/pkg/model/miscutils.go [hunk 2]
```diff
@@ -48,7 +48,6 @@ func (b *MiscUtilsBuilder) Build(c *fi.NodeupModelBuilderContext) error {
 		packages = append(packages, "wget")
 		if b.Distribution != distributions.DistributionAmazonLinux2023 {
```
[the remark this answers is not in the document]
Comment (jsafrane) [reply nodeup/pkg/model/miscutils.go]: done

**Comment by jsafrane** [comment]
/test pull-kops-aws-distro-rhel8
/test pull-kops-aws-distro-rocky8
just to be sure...

**Pull request merged**